ipi/engine/forcefields.py

```diff
@@ -1127,46 +1127,37 @@ def gather(self, r):
             r["result"][1] = final_frc
             r["result"][2] = final_vir
 
-        for i_r in range(ncommittee):
-            if self.extras_mode == "light":
-                ff_res.append(
-                    {
-                        "v": rescaled_pots[i_r],
-                    }
-                )
-            elif self.extras_mode == "heavy":
-                ff_res.append(
-                    {
-                        "v": rescaled_pots[i_r],
-                        "f": list(rescaled_frcs[i_r]),
-                        "s": list(rescaled_virs[i_r].flatten()),
-                        "x": xtrs[i_r],
-                    }
-                )
-
-        r["ff_results"] = ff_res
+        for i_r in range(ncommittee):            
+            committee_results.append(
+                {
+                    "v": rescaled_pots[i_r],
+                    "f": list(rescaled_frcs[i_r]),
+                    "s": list(rescaled_virs[i_r].flatten()),
+                    "x": xtrs[i_r],
+                }
+            )
+            
+        r["result"][3] = {
+            "committee_pot" : rescaled_pots,
+            "committee_force" : rescaled_frcs.reshape(len(rescaled_pots),-1),
+            "committee_virial" : rescaled_virs.reshape(len(rescaled_pots),-1),
+            }
```

Comment:
shall we also add a "committee_weight" ? May be beneficial for a user to have direct access to it.